src/adaptation/icap/ModXact.cc

```diff
@@ -42,8 +42,6 @@
 CBDATA_NAMESPACED_CLASS_INIT(Adaptation::Icap, ModXact);
 CBDATA_NAMESPACED_CLASS_INIT(Adaptation::Icap, ModXactLauncher);
 
-static const size_t TheBackupLimit = BodyPipe::MaxCapacity;
```

Comment:
I am against this removal because it obscures (currently self-documented) logic behind TheBackupLimit-using code. It may also complicate migration to smarter/flexible limits if we ever decide to change this logic. Can we do something like this instead?

```C++
constexpr auto TheBackupLimit = BodyPipe::MaxCapacity;
```

AFAIK, `static` is implied for namespace-scope constants, so we do not need it here, but, if you prefer, we can keep it:

```C++
static constexpr auto TheBackupLimit = BodyPipe::MaxCapacity;
```

Reply:
Hmm, IMO this would be better as a `using`/`typedef` no?

```
using TheBackupLimit = BodyPipe::MaxCapacity;
```